packages/eas-cli/src/commands/deploy/index.ts

```diff
@@ -65,10 +65,9 @@ interface DeployInProgressParams {
 }
 
 export default class WorkerDeploy extends EasCommand {
-  static override description = 'Deploy your Expo web build';
-  static override aliases = ['deploy'];
```

Comment:
We're missing a `worker:deploy` alias here, right? That one's maybe less important, but for completeness I'd add it.